third_party/googletest/README.google

```diff
@@ -1,5 +1,5 @@
-URL: https://github.com/google/googletest/archive/release-1.10.0.tar.gz
-Version: 1.10.0
+URL: https://github.com/google/googletest/archive/e2239ee6043f73722e7aa812a459f54a28552929.zip
+Version: 2021.06.11
```

Comment:
These 2-line changes in this file is the only thing I edited manually. The rest of the commit is a verbatim copy of the upstream Googletest repository.